src/runtime/rpc/rpc_endpoint.h

```diff
@@ -147,8 +146,7 @@ class RPCEndpoint {
    * \param ctx_from The source context.
    * \param type_hint Hint of content data type.
    */
-  void CopyFromRemote(void* from, size_t from_offset, void* to, size_t to_offset, size_t nbytes,
-                      TVMContext ctx_from, DLDataType type_hint);
+  void CopyFromRemote(DLTensor* from, void* to_bytes, uint64_t nbytes);
```

Comment:
All over we use mutable `from` tensor, is there a good reason?

Reply:
This was discussed in the initial proposal. What we really wanted was a DLTensor with const void* in the data field. Making const DLTensor* won't provide much protection as expected, other than the syntax highlighting the fact that from should be constant.

Eventually I think DLTensor* is fine here to be consistent with https://github.com/apache/tvm/blob/main/include/tvm/runtime/c_runtime_api.h#L425

Reply:
Agreed on const void* for data field protection. Prefer to provide as much direction as possible on ownership, but there's only so much one can indicate with c pointers.